demos/react/ContextDemo.tsx

```diff
@@ -4,13 +4,16 @@
  */
 
 import React, { useState } from 'react';
+
```

Comment:
`demos/react` doesn't make much sense.

Should we rename these demos to `cdn-multiroot`, `cdn`, and `npm`?